wren-ui/src/apollo/server/repositories/modelRepository.ts

```diff
@@ -22,4 +24,8 @@ export class ModelRepository
   constructor(knexPg: Knex) {
     super({ knexPg, tableName: 'model' });
   }
+  findAllByIds = async (ids: number[]) => {
```

Comment:
add `public` to method.

Reply:
delivered in 511d5faf675f40bd516a00d5258178530bae5d5b